src/cascadia/TerminalApp/ActionArgs.h

```diff
@@ -330,6 +331,38 @@ namespace winrt::TerminalApp::implementation
         }
     };
 
+    struct SetColorSchemeArgs : public SetColorSchemeArgsT<SetColorSchemeArgs>
+    {
+        SetColorSchemeArgs() = default;
+        GETSET_PROPERTY(winrt::hstring, SchemeName, L"");
+
+        static constexpr std::string_view NameKey{ "name" };

```

Comment:
should it be `colorScheme` for equivalence with the profile?

Reply:
It absolutely should